runtime/vm/CMakeLists.txt

```diff
@@ -21,23 +21,50 @@
 ################################################################################
 
 omr_add_tracegen(j9vm.tdf)
-j9vm_add_library(j9vm SHARED
-	OUTPUT_NAME j9vm${J9VM_VERSION_SUFFIX}
+
+set(interp_flags_to_remove) # list of flags removed when compiling the interpreter
+set(interp_new_flags) # flags to be added when compiling the interpreter
+
+if(OMR_ARCH_POWER)
+	if(OMR_TOOLCONFIG STREQUAL "xlc")
+		list(APPEND interp_flags_to_remove "-O3")
+		list(APPEND interp_new_flags "-O2" "-qdebug=lincomm:ptranl:tfbagg")
+		if(OMR_OS_LINUX)
```

Comment:
I don't think we support any Linux compiling the interpreter with xlc.

Reply:
It is in the codebase, although I don't know if it actually works or not (https://github.com/eclipse/openj9/blob/2020312e3247a65fa6e1e9dbe405d4a67f19bf62/runtime/makelib/targets.mk.ftl#L566)

Reply:
We can never get to this if statement, can we?  It's enclosed in a if power && AIX check

Reply:
See above comment about bracket comments.  
 I decided to leave code in, along with explanation at top of if block, 
 to closer reflect the uma builds

Reply:
Can you add a comment to explain the reason for this code to be here, and that it's commented out?  Having that expressed in the code will make it easier to understand in the future